velox/vector/tests/VectorMaker.cpp

```diff
@@ -220,6 +220,39 @@ ArrayVectorPtr VectorMaker::arrayOfRowVector(
       0);
 }
 
+ArrayVectorPtr VectorMaker::createArrayVector(
+    const std::vector<vector_size_t>& offsets,
+    const std::vector<vector_size_t>& sizes,
+    const VectorPtr elementVector,
+    const std::vector<vector_size_t>& nulls) {
+  BufferPtr offsetsBuffer = allocateOffsets(offsets.size(), pool_);
```

Comment:
nit: perhaps, check that number of elements in offsets, sizes and nulls match.

Reply:
Offsets and sizes should match, nulls need not match as I envision generally nulls will be sparser ; Thus the nulls being passed as a vector so its easy to specify so and so rows are null.